instrumentation/afl-compiler-rt.o.c

```diff
@@ -308,7 +323,7 @@ static void __afl_map_shm_fuzz() {
     }
 
     map =
-        (u8 *)mmap(0, MAX_FILE + sizeof(u32), PROT_READ, MAP_SHARED, shm_fd, 0);
+        (u8 *)mmap(0, shm_fuzz_page_size, PROT_READ, MAP_SHARED, shm_fd, 0);
```

Comment:
If this represents the length of input, why not just name it something like `AFL_SHM_FUZZ_FILE_SIZE`? The page size might be confusing and potentially suggest to users that it should be only the host page size.